src/mongoc/mongoc-uri.c

```diff
@@ -990,6 +1046,15 @@ mongoc_uri_unescape (const char *escaped_string)
 }
 
 
+const mongoc_read_prefs_t *
+mongoc_uri_get_read_prefs_t (const mongoc_uri_t *uri) /* IN */
```

Comment:
Noted. I'll add this deprecation and the update the documentation in this PR when I get a chance.